src/Common/StringSearcher.h

```diff
@@ -569,8 +569,10 @@ class StringSearcher<true, ASCII> : public StringSearcherBase
     uint8_t first{};
 
 #ifdef __SSE4_1__
-    /// vector filled `first` for determining leftmost position of the first symbol
-    __m128i pattern;
+    uint8_t second{};
```

Comment:
Let's add a comment before l. 573.: `/// second character of "needle" (if it's length is > 1)`.

Reply:
Done.